Gruntfile.js

```diff
@@ -73,9 +75,39 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-curl');
     grunt.loadNpmTasks('grunt-zip');
     grunt.loadNpmTasks('grunt-file-creator');
-    grunt.loadNpmTasks('grunt-npm-install');
     grunt.loadNpmTasks('grunt-webpack');
 
+    grunt.registerTask('npm-install', 'Install plugin NPM dependencies', function (plugin) {
+        var done = this.async();
+
+        var target = path.resolve('node_modules_' + plugin);
+
+        if (fs.existsSync(target + '/node_modules')) {
+            grunt.log.writeln('NPM dependencies for plugin ' + plugin + ' already installed.');
+            done();
+        }
+
+        var npm = require('npm');
+        var npmConf = {
+            prefix: target,
+            g: true
+        };
+        var errorHandler = require('npm/lib/utils/error-handler');
+
+        var modules = Array.prototype.slice.call(arguments, 1);
```

Comment:
`arguments.slice` works for me in the browser, didn't test it in node though. If it works in node v4+, we should do it.